dependabot.yml

```diff
@@ -0,0 +1,39 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
This file should be under `.github` folder, I think. [Here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#about-the-dependabotyml-file) is the guide.